ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3913,6 +3913,10 @@ tests:
     env:
       E2E_RUN_TAGS: '@amd64 and @vsphere-ipi and @network-ovnkubernetes and not @fips'
       TAG_VERSION: '@4.14'
+    leases:
```

Comment:
Before it only need one lease assigned by prow ci from `vsphere-multizone-2`, now it needs two more?

Reply:
Yes, previously we hardcode the rest of two segments(ocp-ci-seg-20/ocp-ci-seg-21) in install-config.